www/docs/further/faq.mdx

```diff
@@ -36,6 +36,7 @@ Here are some things to check:
 - Make sure you have the same version of all `@trpc/*` across all your project
 - Make sure you have `"strict": true` in all your `tsconfig.json`s
 - Make sure you have no type errors in your app
+- Make sure you have `"paths": [...]` in your client `tsconfig.json` like your server `tsconfig.json`,so that the client can find the same file
```

Comment:
Not everyone has scoped `tsconfig` files and in the case of bundled monorepo packages this wouldn't be an issue so I would say we need to clarify that in the scenario of multiple tsconfigs this might be an issue.

```suggestion
- In the case that you have a dedicated server and client `tsconfig.json` files without a bundled server monorepo package, make sure you have `"paths": [...]` in your client `tsconfig.json` like your server `tsconfig.json`,so that the client can find the same file.
```